### DISCUSSION TITLE
Keyboard Navigation / A11y

### DISCUSSION THREAD

**Discussion by moklick** [xyflow/xyflow#1625] [New Features]
Keyboard navigation and a better accessibility would be very helpful. Input: #1033

**Comment by shaliniM12** [comment]
Hi, Is there any update on this?

**Comment by tscislo** [comment]
I did some follow-up here: https://github.com/wbkd/react-flow/discussions/2675

**Comment by moklick** [comment]
Since v11 we have better a11y defaults: https://reactflow.dev/docs/guides/accessibility/
